### DISCUSSION TITLE
[Feature Request] Selector and URLTest for Routing Nodes

### DISCUSSION THREAD

**Discussion by Revincx** [immortalwrt/homeproxy#212] [General]
在自定义路由模式下，给路由节点添加 “类型” 属性，可选择 Selector 和 URLTest 两种类型的路由节点，下方的节点选项可以选择多个节点。

这样可以直接利用 sing-box 的 Selector 和 URLTest 出站，同时也能再当前路由节点所选择的节点不可用的情况下实现自动切换。

**Comment by muink** [comment]
你请求的功能已有 fork https://github.com/muink/homeproxy
需要 ipk 请前往下载 https://fantastic-packages.github.io/packages/

**Comment by Revincx** [comment]
> 你请求的功能已有 fork https://github.com/muink/homeproxy 需要 ipk 请前往下载 https://fantastic-packages.github.io/packages/

感谢，我等会试试

**Comment by menghuu** [comment]
> 你请求的功能已有 fork https://github.com/muink/homeproxy 需要 ipk 请前往下载 https://fantastic-packages.github.io/packages/

@muink 您好，https://github.com/muink/homeproxy 会合并到这个仓库吗？

**Comment by muink** [comment]
@menghuu See #47 & #49

**Comment by LoogCP** [comment]
muink在README里面声称不再更新，现在考虑把muink弄得功能合并到这个仓库吗

**Comment by 1715173329** [comment]
永远不会支持 clash 的任何东西，喜欢 clash 可以直接去用 clash。

**Reply by zpvel** [reply]
> 永远不会支持 clash 的任何东西，喜欢 clash 可以直接去用 clash。

urltest 不是singbox 本身不就支持的吗？

**Comment by LoogCP** [comment]
> 永远不会支持 clash 的任何东西，喜欢 clash 可以直接去用 clash。

并不是喜欢clash，只是想要Selector和URLTest用来自动切换

**Comment by science2468** [comment]
> > 永远不会支持 clash 的任何东西，喜欢 clash 可以直接去用 clash。
> 
> 并不是喜欢clash，只是想要Selector和URLTest用来自动切换

 你去fork，然后自己维护就行了。要不就不用，或者自己直接用sing-box。

**Comment by science2468** [comment]
> muink在README里面声称不再更新，现在考虑把muink弄得功能合并到这个仓库吗

我在muink里面的homeproxy仓库没看到你说的内容。
